docs/how-to/viewing-routers/monitoring.md

```diff
@@ -0,0 +1,5 @@
+## Monitoring
+
+The monitoring tab will bring you to the Grafana dashboard of the router. This dashboard displays information about the standard router metrics such as throughput, latency and error rate, to name a few. 
```

Comment:
This will not work out of the box for external users either (just like alerts) and we are missing sufficient documentation on how they can get the monitoring configured. This too can be captured in our backlog for the time being and removed from Gitbooks until we have better dev docs around this.

Reply:
Alright removing this.